scout/server/blueprints/institutes/templates/overview/institute.html

```diff
@@ -0,0 +1,201 @@
+{% extends "layout_bs4.html" %}
+{% from "overview/utils.html" import users_table%}
+
+{% block title %}
+  {{ super() }} - Institutes
+{% endblock %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/css/bootstrap-select.min.css">
+<link href="https://cdn.jsdelivr.net/npm/select2@4.0.12/dist/css/select2.min.css" rel="stylesheet" />
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('overview.institutes') }}">
+      institutes
+    </a>
+  </li>
+  <li class="nav-item active">
+    <p class="navbar-text text-nowrap">{{ institute.display_name }}</p>
+  </li>
+{% endblock %}
+
+{% block content_main %}
+<form  action="{{ url_for('overview.institute', institute_id=institute.internal_id) }}" method="POST"
+  accept-charset="utf-8" id="institute_form">
+{{ form.csrf_token() }}
+<div class="container">
+  <div class="display:block">
+    <!-- institute settings panel -->
+    <div class="card mt-3">
+      <div style="padding: 0;" class="card-body mt-3 ml-3 mr-3" >
+        <h5 class="header">Institute Settings</h5>
+          <div class="form-row mt-3 d-flex justify-content-between">
+            <div class="form-group col-md-12">
+              {{ form.display_name.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Name of the institite which will be displayed on Scout pages") }}
+              {{ form.display_name(class='form-control', value=institute.display_name) }}
+              {% for error in form.display_name.errors %}
+                <span style="color: red;">{{ error }}</span><br>
+              {% endfor %}
+            </div>
+          </div>
+          <div class="form-row mt-3 mb-3">
+            <div class="col-md-2">
+              {{form.sanger_emails.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Email addresses to send variant verification email to")}}
```

Comment:
...only accepts current Scout user emails: exceptions to this can be made by a db admin. Please ask your support!